### ISSUE TITLE
Handle Spotify authorization revokes

### ISSUE THREAD

**Issue by nukdokplex** [kittinan/spotify-github-profile#88]
![image](https://github.com/user-attachments/assets/bb727eb2-9fbf-4d53-aba3-0f43277ec512)

**Comment by kittinan** [comment]
I will investigate this weekend. 

```
https://spotify-github-profile.kittinanx.com/api/view?uid=k8d2u3l1klp64mdypz4b983mn&redirect=true
```

**Comment by nukdokplex** [comment]
@kittinan i think this story will be helpful. A couple of days ago i logged into my Spotify account on my laptop and Spotify found the attempt suspicious and forced me to change my password. After i did this, all applications (including third-party ones) where i logged into my account were logged out.

![image](https://github.com/user-attachments/assets/a98225d0-fddb-4122-b3cb-de180b6e805f)

I think your application needs to somehow show the fact that the authorization has been revoked. Simply put: "handle this exception".

**Comment by kittinan** [comment]
@nukdokplex thank you for the clue

**Comment by kittinan** [comment]
@nukdokplex I fixed it #89. thank you for your report this issue.

**Comment by nukdokplex** [comment]
@kittinan no problem <3
